code/src/index.css

```diff
@@ -1,13 +1,15 @@
+@font-face{
+  font-family: helpme;
+  src:url(HelpMe.ttf)
+}
```

Comment:
is this an imported fun? if so, fun idea! (can't see it anywhere so hard to know :))